.github/workflows/publish-package-docs.yaml

```diff
@@ -94,3 +94,10 @@ jobs:
           pushd build-directory/docs/docsite
           ln -s ${VERSION} _build/latest
           popd
+
+      - name: Deploy docs build
+        run: |
+          rm -rf /var/www/html/ansible-package-docs
```

Comment:
Why do you feel the need to modify this directory within the GitHub Runner? It seems you're being confused here.

Reply:
Hey Sviat. This probably isn't necessary but I added this to "clean" the directory so there are no residual files from the previous run. I was debating whether to do it or not.

Reply:
In general, the automation shouldn't touch things outside the working directory in GHA at all.

Reply:
OK. I'll remove this line. Cheers.